gsoc_contributors_guide.md

```diff
@@ -0,0 +1,33 @@
+# GSoC Contributor's Guide
+
+The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/) participants to help us create the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the the data exploration and discovery process. This project will go hand-in-hand with ongoing development of the [echopype](https://github.com/OSOceanAcoustics/echopype) package that handles the standardization, pre-processing, and organization of these data. We aim for creating ocean sonar data visualization functionalities based on the [HoloViz](https://github.com/holoviz/holoviz) suite of tools, and developing widgets that can be flexibly combined in [Panel](https://panel.holoviz.org/) or used in Jupyter notebooks for interactive data visuzliation.
+
+
+## Getting Started
+
+TBD
+
+
+
+
+## Brainstorm with us!
+
+In the [Issues](https://github.com/OSOceanAcoustics/echoshader/issues) section of this repository, we list some visualization ideas from mentors. We encourage you as a GSoC participant to propose your own original ideas as new issues.
+
+Feel free to propose project ideas [raising an issue](https://github.com/OSOceanAcoustics/echoshader/issues/new/choose) in this repo.
```

Comment:
Change to "propose project ideas by raising an issue". Also probably good to use the direct link that uses the template right away.